genapi/genapi.go

```diff
@@ -692,11 +706,36 @@ func (g *GenAPI) doLever() {
 	g.Lever.Parse()
 }
 
+func SRVClientPreprocess(m *dns.Msg, dc string) {
```

Comment:
I'm not sure we should make this public yet. Let's wait for something else to actually need to use it and see what exactly they need.